README.md

```diff
@@ -8,30 +8,17 @@
 
 An [Ansible AWX](https://github.com/ansible/awx) operator for Kubernetes built with [Operator SDK](https://github.com/operator-framework/operator-sdk) and Ansible.
 
-<!-- Regenerate this table of contents using https://github.com/ekalinin/github-markdown-toc -->
-<!-- gh-md-toc --insert README.md -->
-<!--ts-->
+This Kubernetes Operator is meant to be deployed in your Kubernetes cluster(s) and can be used to install and manage the lifecycle of an AWX instance in the same namespace.
 
-# AWX Operator Documentation
+## Documentation
 
-The AWX Operator documentation is now available at https://awx-operator.readthedocs.io/
-
-For docs changes, create PRs on the appropriate files in the /docs folder.
+The AWX Operator documentation is now available at <https://awx-operator.readthedocs.io/>
```

Comment:
```suggestion
The AWX Operator documentation is available at <https://ansible.readthedocs.io/projects/awx-operator/en/latest/>
```

Reply:
Changed to `https://ansible.readthedocs.io/projects/awx-operator/` as listed in https://www.ansible.com/ecosystem/